src/ws/session-utils.c

```diff
@@ -388,8 +388,7 @@ fork_session (char **env, int (*session)(char**))
 }
 
 void
-utmp_log (int login,
-          const char *rhost)
+utmp_log (int login)
```

Comment:
Nitpick: If the plan is to eventually put rhost into utmp, should we keep the `rhost` parameter and pass `""`?

Reply:
I thought about this too, but I don't see any trouble to add it back in again, either.  Having a parameter where we pass `""` just because we might pass something else at some point in the future seems a bit silly.

Reply:
Argh, well spotted, thanks! I copied a little too much from the other PR. Fixed!